src/main/java/Date.java

```diff
@@ -0,0 +1,67 @@
+/**
+ *  Represents a timing on a specific date. A <code>Date</code> object corresponds
+ *  to a date and a time. e.g.<code>27/08/2019 1800</code>
+ */
+public class Date {
+    protected int day;
+    protected String mon;
+    protected int yr;
+    protected double time;
+    protected String meridian;
+    private String[] months = {"January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"};
```

Comment:
Interesting way of storing dates. I'm quite curious why you created a Date class, since Java also has a Date class (albeit a flawed one).